raiden/tests/integration/fixtures/raiden_network.py

```diff
@@ -14,6 +14,21 @@
     wait_for_token_networks,
 )
 from raiden.tests.utils.tests import shutdown_apps_and_cleanup_tasks
+from raiden.waiting import wait_for_block_using_web3
+
+
+def wait_for_confirmed_block(blockchain_services, raiden_apps):
+    wait_for_block_using_web3(
+        web3=blockchain_services[2].client.web3,
```

Comment:
why are you using `[2]` here / how are you sure that index is actually defined? maybe you want to use `[-1]` ?

Reply:
`blockchain_services` is a `NamedTuple`, it always has 4 items. It would be better to use the attribute name of course, so I changed it to `blockchain_services.deploy_service.client.web3`.